cmd/opampsupervisor/supervisor/commander/commander_others.go

```diff
@@ -0,0 +1,20 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build !windows
+
+package commander
+
+import (
+	"os"
+	"syscall"
+)
+
+func sendShutdownSignal(process *os.Process) error {
+	return process.Signal(os.Interrupt)
```

Comment:
I think `os.Interrupt` is defined as `SIGINT`. Previously we were sending `SIGTEM`. Does this matter?

Reply:
Good question. I switched to using os.Interrupt/SIGINT here because os.Interrupt and os.Kill are the only signals guaranteed to be present across all systems (https://pkg.go.dev/os#Signal) - in practice, it might be that SIGTERM is defined for all the platforms we care to support, but I figured it doesn't hurt to use the more ubiquitous option here.

The collector treats both these [signals the same](https://github.com/open-telemetry/opentelemetry-collector/blob/ef07ea073562c4eb4d63a9032e84a802d8424fe2/otelcol/collector.go#L298), so I figured it'd be better if we used the more "cross-platform" version.

Reply:
SGTM.